ocs_ci/framework/pytest_customization/reports.py

```diff
@@ -174,3 +234,41 @@ def pytest_report_teststatus(report, config):
         GV.TIMEREPORT_DICT[report.nodeid]["total"] = round(
             GV.TIMEREPORT_DICT[report.nodeid]["total"] + teardown_duration, 2
         )
+
+
+def pytest_logger_config(logger_config):
+    logger_config.add_loggers([""], stdout_level="info")
+    logger_config.set_log_option_default("")
+    logger_config.split_by_outcome()
+    logger_config.set_formatter_class(OCSLogFormatter)
+
+
+@pytest.hookimpl(tryfirst=True)
+def pytest_runtest_setup(item):
+    test_name = f"TEST NAME: {item.name}"
+    console_logger.info(f"\n{separator(symbol_='-', val=test_name)}")
```

Comment:
Is the `\n` new line expected here (when there will be second new line in the output from `separator(...)` function)?

Reply:
yes, in the output I wanted to have one extra empty line as it looked better in the output.